src/components/StartCourseButton.svelte

```diff
@@ -1,23 +1,42 @@
 <script>
+// @ts-nocheck
+  export let course_id
+  export let slug
+  export let status = false
+
   import { mutation } from "svelte-apollo";
   import { START_COURSE } from "$lib/mutations/courses";
 
+
   const startCourse = mutation(START_COURSE);
 
-  async function handleClick() {
+  async function handleStart() {
     try {
       const result = await startCourse({
         variables: {
           user_id: "a4f9150e-4619-4b18-807d-2f273b75c12f",
-          course_id: "6477f941-f072-4f1e-95da-b1d02a81ae6f",
+          course_id: course_id,
```

Comment:
How about you try using the property value shorthand syntax?

Reply:
Good point